### PR TITLE
Implementing contribution for community

### PR DESCRIPTION
**Pull request by Simonbdy**
# PR Checklist 

### _1- PR Comments_
    We are implementing the contribution for the Luos community. This PR is about updating the file *contributing.md*.  

<br><hr/>
*WARNING : Do not edit the checklist below.*  
<hr/><br>

### _2- Developer section_

- [x] [Documentation] is up to date with new feature
- [x] [Tests] are *passed OK* (Non regression, new features & bug fixes)
- [x] [Code Quality] Remind to check if :
    * Each function has a header (description, inputs, outputs) 
    * Code is commented (particularly in hard to understand areas)
    * There are no new warnings that can be corrected
    * Commits policy is respected (constitancy commits, clear commits comments)
<hr/><br>

### _3- QA section_

- [x] [Review] tests for new features have been *reviewed*
- [x] [Changelog] is up-to-date with expected tags  
    🆕 Feature: [Feature] Description...  
    🆕 Added: [Feature] Description...  
    🆕 Changed: [Feature] Description...  
    🛠️ Fix: [Feature] Description...

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,53 +1,151 @@
-<a href="https://www.luos.io"><img src="https://uploads-ssl.webflow.com/601a78a2b5d030260a40b7ad/603e0cc45afbb50963aa85f2_Gif%20noir%20rect.gif" alt="Luos logo" title="Luos" align="right" height="100" /></a>
+<a href="https://luos.io"><img src="https://uploads-ssl.webflow.com/601a78a2b5d030260a40b7ad/603e0cc45afbb50963aa85f2_Gif%20noir%20rect.gif" alt="Luos logo" title="Luos" align="right" height="100" /></a>
```
Comment (K0rdan) [inline CONTRIBUTING.md:1]: Don't use old webflow image

**Review by K0rdan — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,52 +2,150 @@
...
+
+## What is contribution and why contributing?
+
+Contribution to a project is about dedicating own time and resources to help a project grow. It can be done in many different ways that are not all about code programming. Anyone can contribute to a project at their own level.
+
+When done right, contribution brings a lot to a project, but also to the contributor. 
+
+For example, it brings together external visions and skills to build, improve or fix parts of this project. Most of the contributors were first mere users of a project before to become contributors. They often got through contribution in order to add a feature they wanted, or fix an issue that was bothering them. The result is also, on their side, the feeling of being part of a great project.
+
+Contribution allows contributors to learn faster and gain skills. In fact, some contributors in open source projects became quite famous and were able to build their carreer around their contributions. That is whay contribution also brings a feeling of gratification for participating in something big.
```
[the remark this answers is not in the document]
Comment (Simonbdy) [reply CONTRIBUTING.md:36]: `why`, I think!
Thanks

**Comment by Simonbdy** [comment]
@K0rdan, all your modif requests have been done.

File: CONTRIBUTING.md (hunk 1 above)
Comment (K0rdan) [reply CONTRIBUTING.md:1]: ```Luos logo```

**Review by K0rdan — APPROVED** [review]

**Pull request merged**